.github/actions/e2e_cleanup_timeframe/action.yml

```
@@ -8,9 +8,15 @@ inputs:
   encryptionSecret:
     description: 'The secret to use for decrypting the artifacts.'
     required: true
-  azure_credentials:
+  azureCredentials:
```

```suggestion
  azure_credentials:
```
The `login_azure` action uses snake case for its input parameters, let's keep it the same here for consistency.